cli/docs/enrich/help.go

```diff
@@ -0,0 +1,14 @@
+package enrich
+
+import (
+	"github.com/jfrog/jfrog-cli-core/v2/plugins/components"
+	"github.com/jfrog/jfrog-cli-security/cli/docs"
+)
+
+func GetDescription() string {
+	return "Enrich sbom format JSON located on the local file-system with Xray."
+}
+
+func GetArguments() []components.Argument {
+	return []components.Argument{{Name: "source pattern", ReplaceWithFlag: docs.SpecFlag, Description: `Specifies the local file system path of the JSON to be scanned.`}}
```

Comment:
```suggestion
	return []components.Argument{{Name: "source pattern", Optional: true, ReplaceWithFlag: docs.SpecFlag, Description: `Specifies the local file system path of the JSON to be scanned.`}}
```
If you are specifing that you can replace with flag, add `Optional` attribute

Reply:
removed replacwithflag